osgp/platform/osgp-domain-core/src/main/java/org/opensmartgridplatform/domain/core/repositories/ProtocolInfoRepository.java

```diff
@@ -16,4 +17,9 @@
 public interface ProtocolInfoRepository extends JpaRepository<ProtocolInfo, Long> {
 
   ProtocolInfo findByProtocolAndProtocolVersion(String protocol, String protocolVersion);
```

Comment:
This causes errors when called with variants in the database

Reply:
Use case doesn't involve variants, exception is ok in this case ("shouldn't happen")